packages/eslint/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 RollupJS Plugin Contributors (https://github.com/rollup/plugins/graphs/contributors)
```

Comment:
```suggestion
Copyright (c) 2020 RollupJS Plugin Contributors (https://github.com/rollup/plugins/graphs/contributors)
```

Reply:
I'd avoid mentioning 2020, let's skip straight to 2021